source/iostreams.tex

```diff
@@ -10544,7 +10544,7 @@
 The \defn{parent directory} of a directory is
 the directory that both contains a
 directory entry for the given directory and is represented by the filename
-\grammarterm{dot-dot} in the given directory.
+dot-dot in the given directory.
```

Comment:
Should we add a forward reference to the definition or "dot-dot", given that it occurs after this point?

Reply:
Good idea.